adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
I am not going to change this to a division. As with other sensors, I use multiplying as it tends to be faster. Particularly when using sensors that can use fast data rates. Here it's used for consistency.